celeryutils/persist_on_failure.py

```diff
@@ -0,0 +1,51 @@
+"""
+Celery utility code for persistent tasks.
+"""
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from celery import Task
+
+from .models import FailedTask
+
+
+# pylint: disable=abstract-method
```

Comment:
disable-abstract-**method** seems like an odd pylint warning to have above a class. I'm sure it's correct, but it's still weird.